p2p/net/udp.go

```diff
@@ -259,12 +266,13 @@ func (n *UDPNet) listenToUDPNetworkMessages(listener net.PacketConn) {
 			mconn := newMsgConnection(conn, n, pk, ns, n.config.MsgSizeLimit, n.config.DialTimeout, n.logger)
 			n.publishNewRemoteConnectionEvent(mconn, node.NewNode(pk, net.ParseIP(host), 0, uint16(iport)))
 			n.addConn(addr, conn)
-			go mconn.beginEventProcessing()
+			go mconn.beginEventProcessing(context.TODO())
```

Comment:
Same here, it's fine to leave `TODO` if you think we should create an app-level context and pass it in here, just wanted to know your thoughts. In general, I think we should use/pass contexts more broadly and we should probably do it as part of the graceful shutdown work (using Tomb).

Reply:
cleaned this one up too. agree re: graceful shutdown work, context objects can be used for that too, and this work should make that easier, but for now I'm focused just on logging.

Reply:
cleaned up as in it's no longer a TODO